recipe/meta.yaml

```diff
@@ -213,9 +213,10 @@ about:
   license: BSD 3-Clause AND Python Software Foundation License
```

Comment:
The license should be `BSD-3-Clause AND PSF-2.0`

Reply:
Thank you - I updated the license